tests/lib/__init__.py

```diff
@@ -1188,6 +1188,7 @@ def create_basic_sdist_for_package(
     *,
     fails_egg_info: bool = False,
     fails_bdist_wheel: bool = False,
+    setup_args: Optional[Dict[str, Any]] = None,
```

Comment:
It’d be better to expose individual arguments you actually need, instead of exposing setuptools’s argument names directly (since that can potentially change). `create_basic_wheel_for_package` uses `depends` to specify dependencies.

Reply:
Okay, I changed the parameter to be the same as in `create_basic_wheel_for_package`.